src/components/laser-controls.js

```diff
@@ -16,21 +16,33 @@ registerComponent('laser-controls', {
     el.setAttribute('gearvr-controls', {hand: data.hand});
     el.setAttribute('oculus-touch-controls', {hand: data.hand});
     el.setAttribute('vive-controls', {hand: data.hand});
+    el.setAttribute('windows-motion-controls', {hand: data.hand});
 
-    // Wait for controller to connect before
-    el.addEventListener('controllerconnected', function (evt) {
+    // Wait for controller to connect, or have a valid pointing pose, before creating ray
+    el.addEventListener('controllerconnected', function (evt) { createRay(evt); });
+    el.addEventListener('controllerdisplayready', createRay);
+
+    function createRay (evt) {
       var controllerConfig = config[evt.detail.name];
 
       if (!controllerConfig) { return; }
 
-      el.setAttribute('raycaster', utils.extend({
+      var raycasterConfig = utils.extend({
         showLine: true
-      }, controllerConfig.raycaster || {}));
+      }, controllerConfig.raycaster || {});
+
+      if (evt.detail.name === 'windows-motion-controls') {
+        var motionControls = el.components['windows-motion-controls'];
+        raycasterConfig = utils.extend(raycasterConfig, motionControls.rayOrigin);
```

Comment:
Can we pass the `rayOrigin` in the `evt.detail` object? This way we don't have to check for `windows-motion-controls` We should maybe move all the controllers to use a `controllermodelready` event for consistency. Maybe better in a separate PR.

Reply:
I like the pattern of passing the ray details in the event, will modify to follow that pattern. Happy to also modify the others, just let me know if you would prefer that in a separate PR once this is completed.

Reply:
Thanks! We can do it in a separate PR so we move things along.